src/callbacks_step/stepsize_dg1d.jl

```diff
@@ -33,15 +33,14 @@ function max_dt(u, t, mesh::TreeMesh{1},
                 constant_diffusivity::False, equations,
                 equations_parabolic::AbstractEquationsParabolic,
                 dg::DG, cache)
-    # to avoid a division by zero if the speed vanishes everywhere,
-    # e.g. for steady-state linear advection
+    # to avoid a division by zero if the diffusivity vanishes everywhere
     max_scaled_speed = nextfloat(zero(t))
 
     @batch reduction=(max, max_scaled_speed) for element in eachelement(dg, cache)
         max_lambda1 = zero(max_scaled_speed)
         for i in eachnode(dg)
             u_node = get_node_vars(u, equations, dg, i, element)
-            lambda1, = max_diffusivity(u_node, equations_parabolic)
+            lambda1 = max_diffusivity(u_node, equations_parabolic)
```

Comment:
I realize this wasn't introduced in this PR, but could we rename `lambda1` to `lambda_diffusive` or something more descriptive? 

It seems like this naming convention expected `lambda2` but we currently don't have that implemented anywhere.

Reply:
Great, thanks!